RNScreenshotDetector.podspec

```
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'RNScreenshotDetector'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/iamacup/react-native-screenshot-detector.git", :tag => "v#{s.version}" }
```

Shouldn't this point to the original author?
```suggestion
  s.source       = { :git => "https://github.com/blend/react-native-screenshot-detector.git", :tag => "v#{s.version}" }
```